rust/cubesql/cubesql/src/compile/snapshots/cubesql__compile__tests__pg_type_is_visible.snap

```diff
@@ -0,0 +1,61 @@
+---
+source: cubesql/src/compile/mod.rs
+expression: "execute_query(\"\n                SELECT t.oid, pg_catalog.pg_type_is_visible(t.oid) isv\n                FROM pg_catalog.pg_type t\n                ORDER BY t.oid ASC;\n                \".to_string(),\n            DatabaseProtocol::PostgreSQL).await?"
+---
++-------+-------+
```

Comment:
Could you please add the type name to this query? It will be much easier to understand the test logic from testing fixtures. Thanks

Reply:
👍